metagpt/actions/design_api.py

```diff
@@ -229,50 +229,21 @@ async def run(self, with_messages, format=CONFIG.prompt_format):
 
     async def _new_system_design(self, context, format=CONFIG.prompt_format):
         prompt_template, format_example = get_template(templates, format)
+        format_example = format_example.format(project_name=CONFIG.project_name)
```

Comment:
I think AGENT will seed the project name from the prompt-context rather than config.

Reply:
That feature is still present; now, an additional function has been added to allow users to specify a project name.